uber/templates/base.html

```diff
@@ -67,7 +67,10 @@
                 {Schedule: [
                     {'View Schedule': '../schedule/'},
                     {'Edit Schedule': '../schedule/edit'}
-                ]}
+                ]},
+            {% endif %}
+            {% if c.HAS_STATS_ACCESS %}
+                {Statistics: '../summary/'}
```

Comment:
Bold move considering how slow this page is!  Still, after the initial round of performance refactors it's not so bad, so that should be fine.